packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -866,7 +885,7 @@ class Auth {
      * A wrapper method for commerce-sdk-isomorphic helper: loginRegisteredUserB2C.
      *
      */
-    async loginRegisteredUserB2C(credentials: LoginRegisteredUserB2CCredentials) {
+    async loginRegisteredUserB2C(credentials: LoginRegisteredUserCredentialsWithCustomParams) {
```

Comment:
NIT: Can you leave a comment why we added `options` into credentials instead of separate argument?